pages/common/decaffeinate.md

```diff
@@ -0,0 +1,19 @@
+# decaffeinate
+
+> Move your CoffeeScript source to JavaScript using modern syntax.
+
+- Convert a CoffeeScript file to JavaScript:
+
+`decaffeinate {{filename}}.coffee`
+
+- Convert a CoffeeScript v2 file to JavaScript:
+
+`decaffeinate --use-cs2 {{filename}}.coffee`
```

Comment:
Could this be `{{path/to/filename}}` please?

Also the same for the other examples.